tests/TestEntry.cpp

```diff
@@ -158,3 +160,109 @@ void TestEntry::testResolveUrl()
 
     delete entry;
 }
+
+void TestEntry::testResolveUrlPlaceholders()
+{
+    Entry* entry = new Entry();
```

Comment:
You could use a stack variable here. No need for dynamic allocation. Or at least use a QScopedPointer.

Reply:
Done